src/foundry/foundry.js/clientDocuments/actor.d.ts

```diff
@@ -25,10 +29,191 @@ declare global {
+     */
+    rollInitiative(options?: ActorRollInitiativeOptions): Promise<void>;
+
+    /**
+     * Refresh the display of active Tokens for this Actor if ActiveEffects were changed
+     */
+    protected _refreshTokens(): void;
+
+    /**
+     * You are referencing Actor#_data which has been deprecated in favor of Actor#data#_source. Support for this reference will be removed in 0.9.0
+     * @deprecated since 0.8.0
+     */
+    get _data(): this['data']['_source'];
+
+    /**
+     * You are referencing Actor#getOwnedItem(itemId) which is deprecated in favor of Actor#items#get(itemId). Support will be removed in 0.9.0
+     * @deprecated since 0.8.0
+     */
+    getOwnedItem(itemId: string): Item | null;
+
+    /**
+     * You are referencing Actor#createOwnedItem which is deprecated in favor of Item.create or Actor#createEmbeddedDocuments. Support will be removed in 0.9.0
+     * @deprecated since 0.8.0
+     */
+    createOwnedItem(
+      itemData: Parameters<this['createEmbeddedDocuments']>[1],
+      options: Parameters<this['createEmbeddedDocuments']>[2]
+    ): ReturnType<this['createEmbeddedDocuments']>;
+
+    /**
+     * You are referencing Actor#updateOwnedItem which is deprecated in favor of Item#update or Actor#updateEmbeddedDocuments. Support will be removed in 0.9.0
+     * @deprecated since 0.8.0
+     */
+    updateOwnedItem(
+      itemData: Parameters<this['updateEmbeddedDocuments']>[1],
+      options: Parameters<this['updateEmbeddedDocuments']>[2]
+    ): ReturnType<this['updateEmbeddedDocuments']>;
+
+    /**
+     * You are referencing Actor#deleteOwnedItem which is deprecated in favor of Item#delete or Actor#deleteEmbeddedDocuments. Support will be removed in 0.9.0
+     * @deprecated since 0.8.0
+     */
+    deleteOwnedItem(
+      itemData: Parameters<this['deleteEmbeddedDocuments']>[1],
+      options: Parameters<this['deleteEmbeddedDocuments']>[2]
+    ): ReturnType<this['deleteEmbeddedDocuments']>;
+
+    /**
+     * You are referencing Actor.fromToken which is deprecated in favor of TokenDocument#getActor. Support will be removed in 0.9.0
+     * @deprecated since 0.8.0
+     */
+    static fromToken(token: Token): Actor;
+
+    /**
+     * You are referencing Actor.createTokenActor which is deprecated in favor of TokenDocument#getActor. Support will be removed in 0.9.0
+     * @deprecated since 0.8.0
+     */
+    static createTokenActor(_baseActor: unknown, token: Token): Actor;
+  }
+
+  interface ActorRollInitiativeOptions {
+    /**
+     * Create new Combatant entries for Tokens associated with this actor.
+     * @defaultValue `false`
+     */
+
+    createCombatants?: boolean;
+    /**
+     * Re-roll the initiative for this Actor if it has already been rolled.
+     * @defaultValue `false`
+     */
+    rerollInitiative?: boolean;
+
+    /**
+     * Additional options passed to the Combat#rollInitiative method.
+     * @defaultValue `{}`
+     * @see Combat#rollInitiative
```

Comment:
Please don't introduce any `@see` that is not present in foundry.js. We can discuss if we want to do this sometimes but in general, we try to stick as close to foundry as possible and do not add stuff without a strict need for it. If you feel you still want this, let's discuss it in Discord and come to a common conclusion how we want to deal with this in general.